sdk/src/wallet/account/operations/transaction/high_level/send.rs

```diff
@@ -31,13 +31,13 @@ use crate::{
 /// Parameters for `send()`
 #[derive(Debug, Clone, Serialize, Deserialize, Getters)]
 pub struct SendParams {
-    /// Bech32 encoded address
-    #[getset(get = "pub")]
-    address: Bech32Address,
     /// Amount
     #[serde(with = "crate::utils::serde::string")]
     #[getset(get = "pub")]
     amount: u64,
+    /// Bech32 encoded address
+    #[getset(get = "pub")]
+    address: Bech32Address,
```

Comment:
Why change the order?

Reply:
consistency (with the `new` ctor arg list which I also changed to reflect the proposed order by Kevin in #586 )

Reply:
@Dr-Electron do you think amount should be first or was it just an example?
Asking because in the cli wallet we also always accept the address first

Reply:
I don't have a strong opinion on it. But if I'm allowed to decide I prefer having the amount first.

Looks nicer if you `send` `x` token to `address`.

Reply:
I think I actually prefer amount first, more consistent with the way it is in outputs